ONSdigital/census-rm-ops
# rehearsal-sample-changes

Motivation and Context
For rehearsal sample spec

What has changed
Changed rgn to region, have not added new fields: field-coordinator-id, field-officer-id or ce-expected-capacity as not in swagger spec?

How to test?
Also with the other branches in the PR images
There's a suitable sample file in the acceptance test branch
You should be able to use this tool to load a rehearsal-sample file and it should arrive on the database and create a file at the end

https://trello.com/c/CzpZXxpZ/741-update-case-data-model-to-reflect-final-rehearsal-sample-8
CASE API:
https://github.com/ONSdigital/census-rm-case-api/tree/rehearsal-sample-spec
ACTION SCH:
https://github.com/ONSdigital/census-rm-action-scheduler/pull/6/files
CASE-PROCERSSOR:
ONSdigital/census-rm-case-processor#11
& new tests:
https://github.com/ONSdigital/census-rm-acceptance-tests/tree/rehearsal-sample-spec
